### PR TITLE
Security assurance

### PR DESCRIPTION
**Pull request by andy-white-hat**
# Description

* Introducing a set of security assurance pages and expanding on the enterprise features previously included.
* A starter for 10 on some kind of security overview cheat sheet (needs further work)

## See changes at: /enterprise/tech-overview/security/overview

# Type of change
- New document(s)/updating existing

# Reviewers
- Prior to merge I need to ensure a few stakeholders are appropriately bought into this content being made public as it's pivoting away from the NDA gated content on the trust portal. From my perspective having it version controlled and interactive is far better.

### CODE REVIEW

File: docs/enterprise/tech-overview/architecture/overview.md [hunk 1]
```diff
@@ -1,23 +1,20 @@
 ---
-title: "Overview"
-description: "Enterprise pages"
+title: "Architecture Overview"
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/architecture/overview.md:2]:
```suggestion
title: "Architecture overview"
```

File: docs/enterprise/tech-overview/architecture/overview.md [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
+description: "A high level view of Codat platform architecture"
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/architecture/overview.md:3]:
```suggestion
description: "A high-level view of Codat platform architecture"
```

File: docs/enterprise/tech-overview/architecture/overview.md [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
 ---
 
-
-# Architecture 
-
+## Introduction
 The Codat technology system has a [microservice](https://en.wikipedia.org/wiki/Microservices) architecture, this modular structure parallelizes software development and enables a scalable and robust system to be created.
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/architecture/overview.md:7]:
```suggestion
Codat has a [microservice](https://en.wikipedia.org/wiki/Microservices) architecture, this modular structure parallelizes software development and enables a scalable and robust system to be created.
```

File: docs/enterprise/tech-overview/architecture/overview.md [hunk 4]
```diff
@@ -1,23 +1,20 @@
...
 
 There are currently over 100 services that make up the Codat technology infrastructure, each with a separate instance for each of the two environments (integration, production). 
 
-Most importantly, each connection to an external data source is a separate service with the responsibility of handling authentication, authorisation, data fetch and data mapping. Services are configured to automatically scale out to multiple instances in the event of increased load, thus ensuring high levels of availability and performance. 
+Most importantly, each connection to an external data source is a separate service with the responsibility of handling authentication, authorization, data fetch and data mapping. Services are configured to automatically scale out to multiple instances in the event of increased load, thus ensuring high levels of availability and performance.
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/architecture/overview.md:11]: 👍

File: docs/enterprise/tech-overview/architecture/overview.md [hunk 5]
```diff
@@ -1,23 +1,20 @@
...
-Most importantly, each connection to an external data source is a separate service with the responsibility of handling authentication, authorisation, data fetch and data mapping. Services are configured to automatically scale out to multiple instances in the event of increased load, thus ensuring high levels of availability and performance. 
+Most importantly, each connection to an external data source is a separate service with the responsibility of handling authentication, authorization, data fetch and data mapping. Services are configured to automatically scale out to multiple instances in the event of increased load, thus ensuring high levels of availability and performance. 
 
 Load on instances is proactively monitored by Azure, and the engineering team is alerted in the event of unexpected spikes.
 
 ![](/img/enterprise/architecture/architecture.png)
 
-# Hosting and Data Storage
-
+## Hosting and Data Storage
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/architecture/overview.md:17]:
```suggestion
## Hosting and data storage
```

File: docs/enterprise/tech-overview/security/data-security.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Data Security"
+description: "Our data security practices"
+---
+
+## Data At Rest
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/security/data-security.md:6]:
```suggestion
## Data at rest
```

File: docs/enterprise/tech-overview/security/data-security.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+This section details how we store data at Codat and how protection is applied to that storage.
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/security/data-security.md:7]:
```suggestion
This section details how we securely store data.
```

File: docs/enterprise/tech-overview/security/data-security.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+### Azure SQL Databases
+* Databases will be encrypted using Transparent Data Encryption (TDE)
```
Comment (mcclowes) [inline docs/enterprise/tech-overview/security/data-security.md:10]:
```suggestion
* Databases are encrypted using Transparent Data Encryption (TDE)
```


Will be? or are? Applies throughout

**Review by mcclowes — COMMENTED** [review]
Partially reviewed, not complete

**Comment by andy-white-hat** [comment]
@mcclowes I've further evolved this and brought in an extra page to talk about PaaS. 

Discussed with Josephine and we're happy that this will replace the current whitepapers on the trust portal. 

Thanks

**Comment by andy-white-hat** [comment]
Intentionally leaving this off the main menu for now @pzaichkina

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**